packages/manager/apps/pci-rancher/src/pages/listing/RancherTaskMessage.tsx

```diff
@@ -8,12 +8,22 @@ import { RancherTask, RancherTaskType } from '@/api/api.type';
 const RancherTaskMessage = ({ tasks }: { tasks: RancherTask[] }) => {
   const { t } = useTranslation('pci-rancher/listing');
   const tasksMessage = tasks
-    .filter((task) => task?.type === RancherTaskType.RANCHER_DELETE)
-    .map(({ id }) => ({
-      id,
-      type: ODS_MESSAGE_TYPE.info,
-      message: t('rancherStatusDeleting'),
-    }));
+    .filter((task) =>
+      [RancherTaskType.RANCHER_DELETE, RancherTaskType.RANCHER_CREATE].includes(
+        task?.type,
```

Comment:
I think that the optional chaining operator `?` is not necessary here because a `RancherTask` is not declared as possibly `undefined` and, if the `tasks` array is empty, the `filter` method performs no action and returns the empty array.

```suggestion
        task.type,
```